package.json

```
@@ -7,8 +7,8 @@
   "type": "module",
   "scripts": {
     "dev": "node --watch .",
-    "start": "node .",
-    "deploy": "node src/deploy-commands"
+    "start": "node src/deploy-commands & node .",
```

Do we want it to run after the deploy?
```suggestion
    "start": "node src/deploy-commands && node .",
```

or after

```suggestion
    "start": "node src/deploy-commands; node .",
```

---

I think if we use "&" its parallel...